src/graph/FetchVerticesExecutor.cpp

```diff
@@ -133,17 +133,30 @@ void FetchVerticesExecutor::processResult(RpcResponse &&result) {
     std::unique_ptr<RowSetWriter> rsWriter;
     auto uniqResult = std::make_unique<std::unordered_set<std::string>>();
     for (auto &resp : all) {
-        if (!resp.__isset.vertices || !resp.__isset.vertex_schema
-                || resp.get_vertices() == nullptr || resp.get_vertex_schema() == nullptr) {
+        if (!resp.__isset.vertices) {
             continue;
         }
-        auto vschema = std::make_shared<ResultSchemaProvider>(resp.vertex_schema);
+
+        auto *schema = resp.get_vertex_schema();
+        if (schema == nullptr) {
+            continue;
+        }
+
+        std::unordered_map<TagID, std::shared_ptr<ResultSchemaProvider>> tagSchema;
```

Comment:
maybe no need shared_ptr is better

Reply:
It is more convenient to use shared_ptr.

Reply:
the tagSchema just used in current thread, so raw pointer is better about performance.

Reply:
When the getRowReader function is called below, its argument is the shared pointer.